api.planx.uk/modules/gis/service/digitalLand.ts

```diff
@@ -214,30 +214,40 @@ async function go(
   }
 
   // --- FLOODING ---
+  const zoneLookup: Record<string, string> = {
+    "flood.zone.2": "flood.zoneTwo",
+    "flood.zone.3": "flood.zoneThree",
+  };
```

Comment:
I think I'm missing something here sorry - might just be a part of the sequencing.

What we're doing below is mapping the old values (incoming request) to the new values (response sent to Editor).

The data migration is updating all values, why do we need this mapping?

Reply:
Sorry another opaque change/lack of code comment on my part ! These lookups are still necessary outside of actual content migration to correctly map the incoming Planning Data entity's `flood-risk-level` property to our correct data field. 

Previously, our `2`, `3`, `I`, `II`, `II*` data value segments lined up exactly the same as Planning Data's values - but now they don't! See diff below as example - 

Before: `(entity) => entity["flood-risk-level"] === zone.split(".").pop()`
After: `(entity) => entity["flood-risk-level"] === oldZone.split(".").pop()`

It's a bit messy but didn't think of a nicer way to handle this!

Reply:
Ah perfect - that's the bit I'm missing - Planning Data are of course still using `2`, `3` etc 👍 

I was caught out by the `old` in the name here I think in combination with the migrations. Maybe just a comment like this will do?

```ts
// This maps the PlanningData key to the PlanX key
const zoneLookup: Record<string, string> = {
```